data/categories.txt

```diff
@@ -8274,7 +8281,7 @@ uk:^Апартаменти
 vi:^Căn hộ
 zh-Hans:^公寓
 zh-Hant:^公寓
-el:^Διαμερίσματα
+el:^Διαμερίσματα για ενοικίαση
```

Comment:
Do you think we need this and simple ```Διαμερίσματα``` is not enough?
Note that the user will have to type all words to see the category.

Reply:
> If this is too long for the app, we may keep only the short version.

The length is not the problem but a synonym is useless if not all of its words are typed in by the user.